src/event/finalization.rs

```diff
@@ -0,0 +1,336 @@
+#![deny(missing_docs)]
```

Comment:
Very nice 💯

Reply:
Yep, saw it somewhere else and decided it was a standard to meet.